variables.tf

```diff
@@ -314,3 +314,13 @@ variable "context_attachments" {
   description = "A list of context IDs to attach to all stacks administered by this module"
   default     = []
 }
+
+variable "context_filters" {
+  type        = map(list(string))
+  description = "Filter stacks using `environments`, `tenants`, or `stages`."
+  default = {
+    tenants      = []
```

Comment:
The default value is not backwards compatible.
If the list is empty, then `lookup(var.context_filters, "tenants"` in 

```
if contains(lookup(var.context_filters, "tenants", [v.vars.tenant]), v.vars.tenant)
```

will return the empty list.

The default should be:

```
default = {
}
```
Note that `lookup` will find `null` as well and return it if we use `tenants = null`.

Or the formula can to be changed:

```
if (lookup(var.context_filters, "tenants", null) == null ||
contains(lookup(var.context_filters, "tenants", [v.vars.tenant]), v.vars.tenant))
......
```

So when
```
default = {
}
```

or 

```
default = {
   tenants = null
}
```

It will be backward compatible with the current deployments (don't filter out anything).


When
```
default = {
    tenants      = []
```

will mean filter out all tenants

Reply:
Thanks @aknysh . I tested it with and without context_filter to make sure it returns the appropriate results.